theme/theme.go

```diff
@@ -98,6 +98,11 @@ const (
 	// Since 2.0.0
 	ColorNameShadow fyne.ThemeColorName = "shadow"
 
+	// ColorNameEntryBackground is the name of theme lookup for entry background color.
+	// TODO or ColorNameEntryFill?
+	// Since 2.0.0
+	ColorNameEntryBackground fyne.ThemeColorName = "entryBackground"
```

Comment:
We try to go for the generic naming from Material, rather than being widget specific.
I think on the call yesterday was saw that MD uses FocusColor for entry background, so my vote gets that.
We already defined this name in some areas, but it was not fully, or correctly, implemented. So we can fix that here.

Reply:
Oh yes, I forgot about it. However the obvious first step must be update the FocusColor with right value, isn't it? Otherwise, the entry will not have any form (just a blue rectangle).